src/types/typesCompat.ts

```diff
@@ -23,4 +23,4 @@ export type Vue2 = typeof Vue2Type
 export type Vue3 = Vue3Type
 export type Vue = Vue3 | Vue2
 
-export type InstallFunction<TOptions> = (vue: unknown, options: TOptions) => void
+export type InstallFunction<TOptions> = (vue: unknown, options?: TOptions) => void
```

Comment:
Just remove `TOptions` completely. `fluent-vue` no longer uses options parameter of `app.use`.